src/components/side-panel/layer-panel/text-label-panel.js

```diff
@@ -51,7 +52,7 @@ export default class TextLabelPanel extends Component {
     const {updateLayerTextLabel, textLabel, fields} = this.props;
     const currentFields = textLabel.map(tl => tl.field && tl.field.name).filter(d => d);
     return (
-      <LayerConfigGroup label={'label'} collapsible>
+      <LayerConfigGroup label={'panel.label.label'} collapsible>
```

Comment:
can we find a better naming for this:
```
panel.label.label
```
having label.label is not ideal

Reply:
That is true. I changed `panel.label...` to `panel.text...`.